staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -425,6 +425,10 @@ type Devices struct {
 	// Defaults to true.
 	// +optional
 	AutoattachMemBalloon *bool `json:"autoattachMemBalloon,omitempty"`
+	// Whether to attach an Input Device.
+	// Defaults to false.
+	// +optional
+	AutoattachInputDevice *bool `json:"autoattachInputDevice,omitempty"`
```

Comment:
it is a bit weird that all our other auto attach booleans default to true but this one defaults to false.

Reply:
Apologies I didn't address this comment directly at the time but while it is different to the others I still think it's valid.